website/docs/r/wafv2_web_acl_logging_configuration.html.markdown

```diff
@@ -14,6 +14,12 @@ Creates a WAFv2 Web ACL Logging Configuration resource.
 If you are capturing logs for Amazon CloudFront, always create the firehose in US East (N. Virginia).
 Be sure to give the data firehose, cloudwatch log group, and/or s3 bucket a name that starts with the prefix `aws-waf-logs-`.
 
+-> **Note:** When logging from a WAFv2 Web ACL to a CloudWatch Log Group the WAFv2 service attempts to create/update a
+generic Log resource policy with a name `AWSWAF-LOGS`, if there are a large number of Web ACLs or the account frequently
+creates and destroys Web ACLs this policy will hit the max policy size and this resource type will fail to be
+created(Full details in this [issue](https://github.com/hashicorp/terraform-provider-aws/issues/25296)). To avoid this
```

Comment:
Minor restructuring of the parenthesis phrasing here.

```suggestion
created (more details can be found in [this issue](https://github.com/hashicorp/terraform-provider-aws/issues/25296)). To avoid this
```